### PR TITLE
Add acknoledgments to README.md and include AUTHORS.txt

### PR DESCRIPTION
**Pull request by sergiorg-hpc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# HPC Team Pre-Commit Hooks
+
+Hooks for pre-commit used by the HPC Team of the Blue Brain Project.
```
Comment (sergiorg-hpc) [inline README.md]: The description is a bit weak, but I honestly do not understand what this repo is about.

Pinging @tristan0x for clarifications.

**Review by matz-e — APPROVED** [review]
Seems good enough to me. It's fairly generic from the looks of it.

File: README.md (hunk 1 above)
Comment (tristan0x) [reply README.md]: I tried to clarify what this repo is about.

Comment (sergiorg-hpc) [reply README.md]: Thank you, @tristan0x and @matz-e!

**Pull request merged**